build.gradle

```diff
@@ -228,7 +228,7 @@ dependencies {
             exclude module: "commons-io"
             exclude module: "log4j-core"
         }
-        compile("com.github.GTNewHorizons:SpongeMixins:1.3.3:dev")
+        compile("com.github.GTNewHorizons:SpongeMixins:1.4.0:dev")
```

Comment:
@SinTh0r4s we should upstream this unless it would cause issues

Reply:
We certainly should upstream this! A manual change in build.gradle is not adived, since the whole file will be replace when an update is rolled out. Wasn't aware of that update. I'll check it and provide an update to the "upstream"

Reply:
It's a new change to support a feature here of identifying mods loaded in dev :-D

Reply:
Also, there's no way to get this working without this change.  Once it's been merged (on sponge mixins), and here, we can upstream the change.  For now everything else will continue working with 1.3.3+